addons/account/models/account_move.py

```diff
@@ -2661,6 +2661,7 @@ class AccountMoveLine(models.Model):
         check_company=True,
         tracking=True)
     account_internal_type = fields.Selection(related='account_id.user_type_id.type', string="Internal Type", store=True, readonly=True)
+    account_internal_group = fields.Selection(related='account_id.user_type_id.internal_group', string="Internal Group", store=True, readonly=True)
```

Comment:
Not sure it's a good idea to add that. It makes a second field to recompute on a lof of move lines (all the ones made on that account) when we change the type of an account :/ Just one is already pretty annoying for people, so... can't we avoid that? We could probably make it non-stored.

Reply:
It will almost not change anything compared to `account_internal_type`, computed and stored at the same time, and if one changes, the other will probably need to change too.

Reply:
But indeed, it was removed here, and added back with version 13.4. I'll see what I can do, if I can remove the other one too.